lib/cocina/models/validators/date_time_validator.rb

```diff
@@ -76,7 +76,8 @@ def valid_edtf?(value)
           #
           # So we catch the false positives from the upstream gem and allow
           # this pattern to validate
-          /\AY-?\d{5,}\Z/.match?(value)
+          /\AY-?\d{5,}\Z/.match?(value) ||
+            /\A\-?\d{1,3}\Z/.match?(value) # temporarily allow format violations
         end
```

Comment:
i'm not positive i fully understand this regex, shouldn't we allow 4 digits too?

Reply:
4 digits is the spec, not an exception.

Reply:
ahh, of course